neqo-crypto/tests/ext.rs

```diff
@@ -6,16 +6,20 @@
 
 use std::{cell::RefCell, rc::Rc};
 
+use handshake::forward_records;
 use neqo_crypto::{
     constants::{HandshakeMessage, TLS_HS_CLIENT_HELLO, TLS_HS_ENCRYPTED_EXTENSIONS},
     ext::{ExtensionHandler, ExtensionHandlerResult, ExtensionWriterResult},
-    Client, Server,
+    generate_ech_keys, AuthenticationStatus, Client, Error, HandshakeState, Server,
 };
-use test_fixture::fixture_init;
+use test_fixture::{damage_ech_config, fixture_init, now};
 
 mod handshake;
 use crate::handshake::connect;
 
+const ECH_CONFIG_ID: u8 = 7;
+const ECH_PUBLIC_NAME: &str = "public.example";
```

Comment:
Do we want to make this configurable?

Reply:
It's just a test.  Isn't a fixed value enough?